bundles/pom.xml

```diff
@@ -43,7 +43,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-scr-plugin</artifactId>
+        <version>1.24.0</version>
```

Comment:
Don't we define the plugin versions in the pluginManagement section of the root pom? So there is one source and different places don't use different stuff?